mungegithub/submit-queue/deployment.yaml

```diff
@@ -42,3 +45,6 @@ spec:
         gitRepo:
           repository: "https://github.com/kubernetes/kubernetes.git"
           revision: "master"
+      - name: cache-volume
```

Comment:
Name this differently -- I want to use this PD for persisting other bits of state beyond just the cache.